doxia-integration-tools/pom.xml

```diff
@@ -131,6 +145,74 @@
       <artifactId>maven-plugin-testing-harness</artifactId>
       <version>1.1</version>
```

Comment:
Maybe go straight to 3.3.0?

Reply:
I think you are mixing up -harness and -tools?

Reply:
Yes, I think you are right.

Reply:
fixed